sources/core/storage/wondows_storage.py

```diff
@@ -0,0 +1,8 @@
+import os
+from sources.core.storage.storage import Storage
+
+
+class WindowsStorage(Storage):
+    def __int__(self):
+        self.base_folder = os.path.join(os.getenv('APPDATA'), 'EncryptMe')
+        os.mkdir(self.base_folder)
```

Comment:
1. No need to create folders here.
 
In general, a constructor is a bad part where we could do any logic. What if we don't have write permissions?

2. os.getenv('APPDATA') returns Roaming folder, right? This folder could be synced with other devices over internet. It's a vulnerability for a private key, even if we made it encrypted in the future. It should be a local folder.